layouts/partials/social-links.html

```diff
@@ -1,60 +1,55 @@
 <nav class="social-links-footer" aria-label='{{i18n "contact-us"}}' >
 
   <span id="contact-us-links"></span>
-    <div class="row equal">
-      <div class="col-sm-4 div-margin" >
-        <div class="border-green">
-          <ul class="ul-margin"><a href="">{{i18n "resources"}}</a></ul>
-          <ul><a href="{{ i18n "accessibility-link" }}">{{i18n "accessibility"}}</a></ul>
-          <ul><a href="">{{i18n "past-projects"}}</a></ul>
-        </div>
-        <div class="border-gray">
-          <ul class="ul-margin"><a href="{{ i18n "roadmap-link" }}">{{i18n "roadmap"}}</a></ul>
-          <ul><a href="{{ i18n "beginning-the-conversation-link" }}">{{i18n "beginning-the-conversations"}}</a></ul>
-        </div>
-
+  <div class="row equal">
+    <div class="col-md-4 div-margin" >
+      <div class="border-green">
+        <ul class="ul-margin"><li><a href="{{ i18n "accessibility-link" }}">{{i18n "accessibility"}}</a></li></ul>
       </div>
-      <div class="col-sm-4 div-margin">
-        <div class="border-brown">
-          <ul class="ul-margin"><a href="">{{i18n "meet-the-team"}}</a></ul>
-          <ul><a href="{{ i18n "our-values-link" }}">{{i18n "our-values"}}</a></ul>
-        </div>
-
+      <div class="border-gray">
+        <ul class="ul-margin"><li><a href="{{ i18n "roadmap-link" }}">{{i18n "roadmap"}}</a></li></ul>
+        <ul><li><a href="{{ i18n "beginning-the-conversation-link" }}">{{i18n "beginning-the-conversations"}}</a></li></ul>
       </div>
-      <div class="col-sm-4 div-margin">
-        <div class="border-yellow">
-          <ul class="ul-margin"><a href="{{ i18n "contact-us-footer-link" }}">{{i18n "contact-us-footer"}}</a></ul>
-          <ul><a href="{{ i18n "newsletter-link" }}">{{i18n "newsletter"}}</a></ul>
-          <ul class="contact-info contact-info-social">
-            <li>
-              <div class="contact-icon">
-                <a title="Twitter" href="https://twitter.com/{{ i18n "twitter" }}">
-                  <i class="icon-twitter"></i>
-                </a>
-              </div>
-            </li>
-            <li>
-              <div class="contact-icon">
-                <a title="LinkedIn"
-                   href="https://linkedin.com/company/{{$.Site.Data.contact.linkedin}}">
-                  <i class="icon-linkedin"></i>
-                </a>
-              </div>
-            </li>
-            <li>
-              <div class="contact-icon">
-                <a title="Github"
-                   href="https://github.com/{{.Site.Data.contact.github}}">
-                  <i class="icon-github"></i>
-                </a>
-              </div>
-            </li>
-          </ul>
-        </div>
 
-      </div>
     </div>
+    <div class="col-md-4 div-margin">
+      <div class="border-brown">
+        <ul class="ul-margin"><li><a href="">{{i18n "meet-the-team"}}</a></li></ul>
```

Comment:
No HREF on this one, should probably go to https://digital.canada.ca/meet-the-team/ ?